examples/expectations/table_expectation_template.py

```diff
@@ -0,0 +1,244 @@
+    }
+
+    metric_dependencies = ("table.alphabetical_column_name_count",)
+    success_keys = ("user_input",)
+
+
+    default_kwarg_values = {
+        "user_input": None,
+        "result_format": "BASIC",
+        "include_config": True,
+        "catch_exceptions": False,
+        "meta": None,
+    }
+
+    def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
+        """
+        Validates that a configuration has been set, and sets a configuration if it has yet to be set. Ensures that
+        necessary configuration arguments have been provided for the validation of the expectation.
+
+        Args:
+            configuration (OPTIONAL[ExpectationConfiguration]): \
+                An optional Expectation Configuration entry that will be used to configure the expectation
+        Returns:
+            True if the configuration has been validated successfully. Otherwise, raises an exception
+        """
+
+        #     # Setting up a configuration
+        try:
+            assert "user_input" in configuration.kwargs, "user_input is required"
+            assert (
+                    isinstance(configuration.kwargs["user_input"], str)
+            ), "user_input must be a string"
+        except AssertionError as e:
+            raise InvalidExpectationConfigurationError(str(e))
+        super().validate_configuration(configuration)
+        return True
+
+    @classmethod
+    @renderer(renderer_type="renderer.prescriptive")
+    @render_evaluation_parameter_string
+    def _prescriptive_renderer(
+            cls,
+            configuration=None,
+            result=None,
+            language=None,
+            runtime_configuration=None,
+            **kwargs
+    ):
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+        params = substitute_none_for_missing(configuration.kwargs, ["value"])
+        template_str = "Must have exactly 4 columns."
+        return [
+            RenderedStringTemplateContent(
+                **{
+                    "content_block_type": "string_template",
+                    "string_template": {
+                        "template": template_str,
+                        "params": params,
+                        "styling": styling,
+                    },
+                }
+            )
+        ]
+
+    # This method will utilize the computed metric to validate that your Expectation about the Table is true
+    def _validate(
+            self,
+            configuration: ExpectationConfiguration,
+            metrics: Dict,
+            runtime_configuration: dict = None,
+            execution_engine: ExecutionEngine = None,
+    ):
+        actual_column_count = metrics.get("table.alphabetical_column_name_count")
+
+        return {
+            "success": actual_column_count == 4,
+            "result": {"observed_value": actual_column_count},
+        }
+
+
+if __name__ == "__main__":
```

Comment:
Does this run?  Should we add a test for it in its present form?..

Reply:
This does - I have a test under the examples section 👍